### PR TITLE
Reword messages to be easier to read.

### PR DESCRIPTION
**Pull request by agatti**
Messages have been reworded to be both easier to read and without typos/grammar errors.

Original messages featured a few grammar errors and were a bit too verbose, so they have been shortened and cleaned up.  Message codes have not been touched.

### CODE REVIEW

**Review by Melevir — CHANGES_REQUESTED** [review]
Cool, thanks a lot! Could you please also change the phrases in readme?

**Comment by agatti** [comment]
Done, I changed all instances and the PR should be (correctly) squashed as well.

**Comment by Melevir** [comment]
Almost there! Fix CI, please.

**Comment by agatti** [comment]
There you go, should pass flake8 checks now.

File: README.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ Tested on Python 3.8+ and flake8 3.9+.
 
 | Error code |                     Description          |
 |:----------:|:----------------------------------------:|
-|   FNE001   | Name of the function says, that is should return `bool`, but it returns *actual_type* |
-|   FNE002   | The method has a `@property` decorator, but has a verb in it's name (*verb*) |
-|   FNE003   | Name of the function uses `save`, but not uses `to` |
-|   FNE004   | Name of the function uses `load`, but not uses `from` |
+|   FNE001   | Function name that it should return a `bool`, but it returns *actual_type* |
```
Comment (Melevir) [inline README.md:49]: Looks like `says` is missing here.

**Review by Melevir — CHANGES_REQUESTED** [review]
Almost there!

**Pull request closed without merging**